frontend/src/pages/ProjectSettings/ProjectFilters/ProjectFilters.tsx

```diff
@@ -195,6 +200,7 @@ export const ProjectProductFilters: React.FC<{
 		},
 	})
 
+	const query = formStore.useValue('exclusionQuery') ?? ''
```

Comment:
i was having issues with this because we actually `setValue` of `exclusionQuery` here based on the loaded previous stored query. needed this here to rerender the query builder after that load happens.